packages/circuit-ui/components/NotificationToast/NotificationToast.tsx

```diff
@@ -119,16 +119,37 @@ const contentStyles = ({ theme }: StyleProps) => css`
 
 const Content = styled('div')(contentStyles);
 
-const StyledIcon = styled.span(
+const IconWrapper = styled.div(
   ({ theme, variant }: StyleProps & { variant: Variant }) =>
     css`
-      display: block;
+      position: relative;
       align-self: flex-start;
       flex-grow: 0;
       flex-shrink: 0;
       line-height: 0;
       color: ${theme.colors[colorMap[variant]]};
     `,
+  // Adds a black background behind the SVG icon to color just the exclamation mark black.
```

Comment:
Did you try to reuse these styles instead of duplicating them across the 3 components? This will make it a bit harder to maintain

Reply:
I looked at it, and I would prefer leaving it as is. The ValidationHint has a slight changes and I think it is not necessary of extracting it as now. If there are similar changes in the future I would rethink the implementation then. I hope that works for you.
I also added the change for the warning text color.

Reply:
Sure that's fine, let's roll with this!